lib/data/models/activity/activity.dart

```diff
@@ -94,34 +93,38 @@ class Activity {
   @IgnoreColumn()
   LatLng get latLng => LatLng(yCoord, xCoord);
 
-  bool isCompleted(){
-
-    switch(activityType){
-      case ActivityType.pictureSelectActivity: {
-        return selectedPictureId != null;
-      }
-      break;
-      case ActivityType.pictureTapActivity: {
-        return userXCoord != null && userYCoord != null;
-      }
-      break;
-      case ActivityType.countActivity: {
-        return userCount != null;
-      }
-      break;
-      case ActivityType.textAnswerActivity: {
-        return userText != null;
-      }
-      break;
-      case ActivityType.photographActivity: {
-        return userPhotoId != null;
-      }
-      break;
-      default: {
-        return false;
-      }
+  bool isCompleted() {
+    switch (activityType) {
+      case ActivityType.pictureSelectActivity:
```

Comment:
The opening brace should be on the same line as the case

Reply:
Dartfmt says otherwise

Reply:
that's weird